syntax = "proto3";

package fbsrankings.messages.query;

message TeamByIDValue {
    string team_id = 1;
    string name = 2;
}

message TeamByIDResult {
    optional TeamByIDValue team = 1;
}

message TeamByIDQuery {
    string query_id = 1;
    string team_id = 2;
}
